specs/altair/beacon-chain.md

```diff
@@ -616,7 +616,10 @@ def process_rewards_and_penalties(state: BeaconState) -> None:
         return
 
     flag_indices_and_numerators = get_flag_indices_and_weights()
-    flag_deltas = [get_flag_index_deltas(state, index, numerator) for (index, numerator) in flag_indices_and_numerators]
+    flag_deltas = [
+        get_flag_index_deltas(state, index, uint64(numerator))
+        for (index, numerator) in flag_indices_and_numerators
+    ]
```

Comment:
Maybe we can update the `get_flag_indices_and_weights()` return type instead, and type the numerator constants?

Reply:
Good idea! Done.
I've changed types of `SYNC_REWARD_WEIGHT` and `WEIGHT_DENOMINATOR` for consistency too.